lib/atomic_web/components/socials.ex

```diff
@@ -0,0 +1,29 @@
+defmodule AtomicWeb.Components.Socials do
+  @moduledoc false
+  use Phoenix.Component
+
+  def socials(assigns) do
+    ~H"""
+    <%= if @entity.socials do %>
+      <div class="mt-4 flex gap-4">
+        <%= for {social, icon, url_base} <- [
+            {:tiktok, "tiktok.svg", "https://tiktok.com/"},
+            {:instagram, "instagram.svg", "https://instagram.com/"},
+            {:facebook, "facebook.svg", "https://facebook.com/"},
+            {:x, "x.svg", "https://x.com/"}
+          ] do %>
+          <% social_value = Map.get(@entity.socials, social) %>
+          <%= if social_value do %>
+            <div class="flex flex-row items-center gap-x-1">
+              <img src={"/images/" <> icon} class="h-5 w-5" alt={social |> Atom.to_string() |> String.capitalize()} />
+              <.link class="text-blue-500" target="_blank" href={url_base <> social_value}>
+                <%= social |> Atom.to_string() |> String.capitalize() %>
```

Comment:
```suggestion
                <%= some_function(social)  %>
```
Please use a function from helpers module for this